src/utils/EquipmentType.ts

```diff
@@ -19,12 +21,17 @@ export const equipmentStyles = {
         alignItems: 'center',
         justifyContent: 'space-between',
     },
-    equipmentTag: (theme: string) => ({
+    equipmentTag: (theme: string | Theme) => ({
         borderRadius: '10px',
         padding: '4px',
         fontSize: 'x-small',
         textAlign: 'center',
-        color: theme === LIGHT_THEME ? 'inherit' : 'black',
+        color:
+            //TODO remove first condition when gridstudy is updated
```

Comment:
Still need this ?

Reply:
yes some fronts like gridstudy pass a string instead on a `Theme` object...